addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -0,0 +1,41 @@
+/* @odoo-module */
+
+import { Chatter } from "@mail/chatter/web_portal/chatter";
+import { Activity } from "@mail/core/web/activity";
+import { SuggestedRecipientsList } from "@mail/core/web/suggested_recipient_list";
+import { RecipientList } from "@mail/core/web/recipient_list";
+import { FollowerList } from "@mail/core/web/follower_list";
+
+import { patch } from "@web/core/utils/patch";
+import { useState } from "@odoo/owl";
+import { usePopover } from "@web/core/popover/popover_hook";
+import { useService } from "@web/core/utils/hooks";
+
+Chatter.components = {
+    ...Chatter.components,
+    Activity,
+    SuggestedRecipientsList,
+    FollowerList,
+};
+
+patch(Chatter.prototype, {
+    setup() {
+        super.setup(...arguments);
+        this.activityService = useState(useService("mail.activity"));
+        this.recipientsPopover = usePopover(RecipientList);
+        this.defaultRequestList = [
+            ...this.defaultRequestList,
+            "followers",
+            "attachments",
+            "suggestedRecipients",
+        ];
```

Comment:
Isn't it equivalent to parent default (message) + requestList() result?
In this case, can't we remove this field completely and replace is by message concat requestList in code?